site/content/contribute/getting-started/core-committers.md

```diff
@@ -1,116 +1,141 @@
     - Dev areas: Security, Plugins, Integrations, Performance
-- **Hyeseong Kim**
+- **<a name="hyeseong.kim">Hyeseong Kim</a>**
     - @cometkim on [community.mattermost.com](https://community.mattermost.com/core/messages/@cometkim) and [@cometkim](https://github.com/cometkim) on GitHub
     - Dev areas: Web App, React, Redux, REST API
-- **Hanzei**
+- **<a name="hanzei">Hanzei</a>**
     - @hanzei on [community.mattermost.com](https://community.mattermost.com/core/messages/@hanzei) and [@hanzei](https://github.com/hanzei) on GitHub
     - Dev areas: Plugins, Integrations, Security
-- **Lev Brouk**
+- **<a name="lev.brouk">Lev Brouk</a>**
     - @lev.brouk on [community.mattermost.com](https://community.mattermost.com/core/messages/@lev.brouk) and [@levb](https://github.com/levb) on GitHub
     - Dev areas: Extensions
-- **Brad Coughlin**
+- **<a name="brad.coughlin">Brad Coughlin</a>**
     - @brad on [community.mattermost.com](https://community.mattermost.com/core/messages/@brad) and [@bradjcoughlin](https://github.com/bradjcoughlin) on GitHub
     - Dev areas: Web App, React, Redux
-- **William Gathoye**
+- **<a name="william.gathoye">William Gathoye</a>**
     - @wget on [community.mattermost.com](https://community.mattermost.com/core/messages/@wget) and [@wget](https://github.com/wget) on GitHub
     - Dev areas: Desktop app, Plugins
-- **Dean Whillier**
+- **<a name="dean.whillier">Dean Whillier</a>**
     - @deanwhillier on [community.mattermost.com](https://community.mattermost.com/core/messages/@deanwhillier) and [@deanwhillier](https://github.com/deanwhillier) on GitHub
     - Dev areas: Desktop app, Web App, React, Redux, UI, React Native
-- **Miguel Alatzar**
+- **<a name="miguel.alatzar">Miguel Alatzar</a>**
     - @miguel.alatzar on [community.mattermost.com](https://community.mattermost.com/core/messages/@miguel.alatzar) and [@migbot](https://github.com/migbot) on GitHub
     - Dev areas: React Native, Desktop app, Web App, React, Redux
-- **Miguel de la Cruz**
+- **<a name="Miguel de la Cruz">Miguel de la Cruz</a>**
     - @miguel.delacruz on [community.mattermost.com](https://community.mattermost.com/core/messages/@miguel.delacruz) and [@mgdelacroix](https://github.com/mgdelacroix) on GitHub
     - Dev areas: Elasticsearch, Web App, Server
-- **Gabe Jackson**
+- **<a name="gabe.jackson">Gabe Jackson</a>**
     - @gabe.jackson on [community.mattermost.com](https://community.mattermost.com/core/messages/@gabe.jackson) and [@gabrieljackson](https://github.com/gabrieljackson) on GitHub
     - Dev areas: Server, Cloud, Kubernetes
-- **Jason Frerich**
+- **<a name="jason.frerich">Jason Frerich</a>**
     - @jason.frerich on [community.mattermost.com](https://community.mattermost.com/core/messages/@jason.frerich) and [@jfrerich](https://github.com/jfrerich) on GitHub
     - Dev areas: Extensions
 
+Core Developers
+---------------
 
 Below is the list of core developers working on individual Mattermost repositories:
```

Comment:
```suggestion
Below is the list of core committers working on individual Mattermost repositories:
```

Reply:
Are these core committers, though? Should I remove the `Core Developers` header and divider?

Reply:
Argh, my brain is in a jumble. I read Jason's comment about them just being core committers for individual repos and I thought you removed the distinction between sections, not added it